CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+### citus v7.0.1 (September 12, 2017) ###
+
+* Fixes a bug causing memory leak in `INSERT ... SELECT` queries
+
+* Fixes a use-after-free bug in prepared statements
```

Comment:
Maybe _Fixes a bug that could cause incorrect execution of prepared statements_?